lib/ansible/modules/cloud/azure/azure_rm_aci.py

```diff
@@ -0,0 +1,404 @@
+        # self.log("master_profile : {0}".format(parameters.master_profile))
+        # self.log("agent_pool_profiles : {0}".format(parameters.agent_pool_profiles))
+        # self.log("vm_diagnostics : {0}".format(parameters.diagnostics_profile.vm_diagnostics))
+
+        credentials = None
+
+        if self.registry_login_server is not None:
+            credentials = ImageRegistryCredential(self.registry_login_server,
+                                                  self.registry_username,
+                                                  self.registry_password)
+
+        ip_address = None
+
+        if self.ip_address is not None:
+                ip_address = IpAddress([self.port], self.ip_address)
+
+        parameters = ContainerGroup(self.location,
+                                    None,
+                                    [Container(self.name, self.image, ResourceRequirements(ResourceRequests(self.memory, self.cpu)))],
+                                    credentials,
+                                    None,
+                                    ip_address,
+                                    self.os_type)
+
+        try:
+            client = ContainerInstanceManagementClient(self.azure_credentials, self.subscription_id)
+            response = client.container_groups.create_or_update(self.resource_group, self.group_name, parameters)
+        except CloudError as exc:
+            self.log('Error attempting to create the container instance.')
+            self.fail("Error creating the ACI instance: {0}".format(str(exc)))
+        return create_aci_dict(response)
+
+    def delete_aci(self):
+        '''
+        Deletes the specified container instance in the specified subscription and resource group.
+        The operation does not delete other resources created as part of creating a container service,
+        including storage accounts, VMs, and availability sets.
+        All the other resources created with the container service are part of the same resource group and can be deleted individually.
+
+        :return: True
+        '''
+        self.log("Deleting the ACI instance {0}".format(self.name))
+        try:
+            client = ContainerInstanceManagementClient(self.azure_credentials, self.subscription_id)
+            response = client.container_groups.delete(self.resource_group, self.group_name)
+        except CloudError as e:
+            self.log('Error attempting to delete the ACI instance.')
+            self.fail("Error deleting the ACI instance: {0}".format(str(e)))
+
+        return True
+
+    def get_aci(self):
+        '''
+        Gets the properties of the specified container service.
+
+        :return: deserialized ACI instance state dictionary
+        '''
+        self.log("Checking if the ACI instance {0} is present".format(self.name))
+        found = False
+        try:
+            response = self.containerinstance_client.container_services.get(self.resource_group, self.name)
```

Comment:
This line won't work because:
- no `self.containerinstance_client` member any more
- `container_services` -> `container_groups`